panel/lxqtpanel.cpp

```diff
@@ -927,13 +941,13 @@ void LXQtPanel::showPopupMenu(Plugin *plugin)
     // Plugin Menu ..............................
     if (plugin)
     {
-        QMenu *m = plugin->popupMenu();
+        QMenu *popupMenu = plugin->popupMenu();
```

Comment:
I don't like this renaming of variables...but if you think it's needed, then, please, make them in separate commit